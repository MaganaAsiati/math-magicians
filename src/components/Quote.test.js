import React from 'react';
import renderer from 'react-test-renderer';
import Quote from './Quote';

test('renders correctly', () => {
  const tree = renderer
    .create(<Quote />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
