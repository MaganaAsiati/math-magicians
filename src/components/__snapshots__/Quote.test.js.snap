// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="block"
>
  <h1
    className="quote"
  >
    "Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding."
  </h1>
  <span
    className="author"
  >
    - William Paul Thurston
  </span>
</section>
`;
